test/utils

```diff
@@ -58,6 +58,10 @@ function curl() {
 export -f curl
 
 function poetry() {
+    if [ "$1" = "export" ] ; then
+        # Fake creating the export file
+        touch requirements.txt.orig
```

Comment:
```suggestion
        # mock creating the export file
        last="$#"
        touch "${!last}"
```

Would this work @felix11h ?

Reply:
I think so! See CI status in https://github.com/moneymeets/python-poetry-buildpack/commit/a7fae83d25b042007638a3e4bce79deabfa30c10.

Reply:
Could you add this, @davegaeddert? Thank you!